src/open_inwoner/accounts/views/contactmoments.py

```diff
@@ -131,7 +130,7 @@ def get_kcm_subject(
                 kcm.contactmoment.url,
                 exc_info=exc,
             )
-            return e_suite_subject_code
+            return ContactFormSubject.objects.first()
```

Comment:
Discussed and request is to split this up, situation with multipleobjectsreturned should show the first of the subjects with the subject_code matching the esuite subject code